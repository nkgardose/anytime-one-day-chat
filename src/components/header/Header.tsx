/** @jsxImportSource @emotion/react */
import React from 'react'
import header from './style'

const Header: React.FunctionComponent<React.PropsWithChildren> = ({ children }) => <nav css={header}>{children}</nav>

export default Header
