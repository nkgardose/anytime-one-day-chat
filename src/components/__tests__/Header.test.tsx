import { render, screen } from '@testing-library/react'
import Header from '../header'
import '@testing-library/jest-dom'
import Providers from '../../providers'

describe('<Header>', () => {
  it('renders properly', () => {
    const { container } = render(
      <Providers>
        <Header>Header</Header>
      </Providers>
    )
    expect(container).toMatchSnapshot()
  })
  it('renders correct children value', () => {
    render(
      <Providers>
        <Header>LGTM Channel</Header>
      </Providers>
    )

    expect(screen.getByText('LGTM Channel')).toBeInTheDocument()
  })
})
