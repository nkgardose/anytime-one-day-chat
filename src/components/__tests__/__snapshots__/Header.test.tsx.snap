// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Header> renders properly 1`] = `
<div>
  <nav
    class="css-17wuva9-Header"
  >
    Header
  </nav>
</div>
`;
